README.md

```diff
@@ -1,96 +1,17 @@
 # GitHub Gems: Driving Open-Source Investments With Data
```

Comment:
How about removing the open-source investments part? This project will be a github data pipeline taht can be used for any purposes not only investment